export const callExternal = async function() {
    try {

    } catch(e) {
        
    }
}